CLUEstering/alpaka/BindingModules/binding_gpu_cuda.cc

```diff
@@ -22,12 +22,13 @@ namespace alpaka_cuda_async {
                                         const std::vector<std::vector<float>>& coords,
                                         const std::vector<float>& weights,
                                         const FlatKernel& kernel,
-                                        int Ndim) {
-	std::vector<Device> devices = alpaka::getDevs<Platform>();
+                                        int Ndim,
+                                        size_t block_size) {
+    std::vector<Device> devices = alpaka::getDevs<Platform>();
```

Comment:
I was thinking that we should allow the users to decide what device they want to use. We could add a method that lists all the available devices for each backend, so that they can choose which to use. What do you think?

Reply:
Yes, but I still think you don't need `std::vector<Device>`, in the line below you select the device with `alpaka::getDevByIdx<Acc1D>(0u);`, right? 

Unless you had something different in mind

Reply:
Yes of course, in that line it's useless. 
I'll write the new function right away.